eng/common/scripts/common.ps1

```diff
@@ -20,12 +20,43 @@ $EngScriptsLanguageSettings = Join-path $EngScriptsDir "Language-Settings.ps1"
 if (Test-Path $EngScriptsLanguageSettings) {
   . $EngScriptsLanguageSettings
 }
-If ($LanguageShort -eq $null)
+if ($null -eq $LanguageShort)
```

Comment:
```suggestion
if ($LanguageShort)
```

Reply:
I don't understand what the idea here. Am trying to test if $LanguageShort is null. Will this suggestion work?

Reply:
This condition will check for null or empty and if it is either of those it will return false. If empty is a valid value then what you have works.

Reply:
Oh I see thanks a lot. In that case it should be `if(-not $LanguageShort)` since am trying to branch into the if statement